src/seabirdscientific/conversion.py

```diff
@@ -334,48 +346,37 @@ def convert_sbe63_oxygen(
     """
     if thermistor_units == "volts":
         temperature = convert_sbe63_thermistor(thermistor, thermistor_coefs)
-    elif thermistor_units == "C":
+    else:  # if thermistor_units == "C":
```

Comment:
Would it be worth leaving the else if and add an else that throws a warning if thermistor_units isn't one of the above?